### ISSUE TITLE
Clicking to purchase a tile works inconsistently

### ISSUE THREAD

**Issue by ubereffect** [Azurency/CQUI_Community-Edition#392]
When I click a city and bring up the city view, I see the purchasable tiles, but actually purchasing them can be a bit of a challenge at times. It seems to take 2 clicks on average, sometimes 3-5 clicks, sometimes 1.

 - OS: Windows 10
 - Version: 1.0.0.290
 - Expansions: All expansions and DLC, including Rise & Fall, and Gathering Storm
 - Other mods: None

I seem to recall this issue being present in CQUI before Gathering Storm, I just shrugged it off as my misclicking. Vanilla is buttery smooth in comparison.

Related to [#216](https://github.com/Azurency/CQUI_Community-Edition/pull/216)?

**Comment by Azurency** [comment]
Yes, it's related, also related to the rewrite in https://github.com/Azurency/CQUI_Community-Edition/commit/6ea61cf772396869cf0fe1bef019c08cacbc10a6#diff-3cebae74c7703440c01145d16ef5ea6cR102 of `plotinfo_CQUI.lua` using `LuaEvents.CQUI_StartDragMap()`

**Comment by Azurency** [comment]
Should be fixed now in the latest version

**Comment by ubereffect** [comment]
Tested it with new Steam Version which includes the fix.

Clicking to purchase tiles worked about 80-90% of the time in my new test. Purchased a 100 or so tiles.  While CQUI was the only mod I used during the test, I used an old saved game file which had a lot of mods associated with it. No issue with fresh game perhaps? Haven't tried that yet.

Even if a fresh game were to give same result. It's A LOT better.

**Comment by Azurency** [comment]
80-90% 👀. Are you sure the 20-10% times it's not working that you're not dragging the map a little ? It should buy the plot when you click on the button unless you're dragging to avoid unwanted purchase.

**Comment by ubereffect** [comment]
Just finished testing it again. Bound mouse click to keyboard this time. If I take my hand of the mouse, before clicking with my keyboard, issue's gone. As for clicking with mouse, I was buying up tiles quickly in previous test. If I'm a little more careful while mouse clicking, it may happen 5% of the time?

Seems very sensitive to mouse drag, which I understand is to prevent accidental tile purchases. Having said that, I'd consider this resolved.
